docs/mobility/motion/constraints.md

```diff
@@ -13,10 +13,11 @@ Constraints are passed as arguments to the [`Move`](../#move) method.
 
 The following constraints are available:
 
-- [Linear Constraint](#linear-constraint)
-- [Orientation Constraint](#orientation-constraint)
+- [Linear constraint](#linear-constraint)
+- [Orientation constraint](#orientation-constraint)
+- [Next steps](#next-steps)
```

Comment:
haha good call

Reply:
I think this might have been the linter doing TOC. I should probably investigate turning that off  😭

Reply:
oh interesting